.eslintrc.js

```diff
@@ -123,13 +125,19 @@ module.exports = {
     "react-native-a11y/has-valid-accessibility-descriptors": 1,
     "react-native-a11y/has-valid-accessibility-ignores-invert-colors": 1,
     "react-native-a11y/has-valid-accessibility-live-region": 1,
-    "react-native-a11y/has-valid-important-for-accessibility": 1
+    "react-native-a11y/has-valid-important-for-accessibility": 1,
+
+    "@typescript-eslint/no-unused-vars": 0,
+    "@typescript-eslint/no-var-requires": 0,
+    "@typescript-eslint/no-empty-function": 0,
```

Comment:
I think this is fine, but you have one-off exceptions to this rule elsewhere in this PR. I'd leave this rule here and remove the one-offs.